partition_tables_templates/rancheros_default_fake.erb

```diff
@@ -0,0 +1,8 @@
+<%#
+kind: ptable
+name: RancherOS default fake
+model: Ptable
+oses:
+- RancherOS
+%>
+# Not required for RancherOS provisioning.
```

Comment:
This might be useful for the future, can you rename the file and rename it to simply "Empty". Then keep the RancherOS association, but do not put any text, it must render to empty string. To make sure there is not even a newline add the minus sign in the end:

```erb
<%#
kind: ptable
name: Empty
model: Ptable
oses:
 - RancherOS
-%>
```

Reply:
In a "empty.erb" file ?